unittests/Basic_tests/DependencyInjection/GrateConfigurationBuilder_Factory.cs

```diff
@@ -128,4 +121,18 @@ public void Creates_default_builder_with_environment_name(string environmentName
         grateConfiguration.Environment.Should().BeEquivalentTo(new GrateEnvironment(environmentName));
     }
     
+    [Fact]
+    public void Creates_default_builder_with_custom_folder_configuration()
```

Comment:
Please check out this test, @JaDuyve  - I think this fix will solve your request, don't you agree?

Reply:
Yes, this should do the trick! Thank you!